server/middleware/reactApplication/ServerHTML.js

```diff
@@ -47,13 +55,18 @@ function ServerHTML(props) {
     <script nonce={nonce} type="text/javascript" dangerouslySetInnerHTML={{ __html: body }} />
   );
 
+  const webpackManifestScript = `
+    window.webpackManifest = ${JSON.stringify(clientWebpackManifest)};
+  `;
+
   const headerElements = removeNil([
     ...ifElse(helmet)(() => helmet.title.toComponent(), []),
     ...ifElse(helmet)(() => helmet.base.toComponent(), []),
     ...ifElse(helmet)(() => helmet.meta.toComponent(), []),
     ...ifElse(helmet)(() => helmet.link.toComponent(), []),
     ifElse(clientEntryAssets && clientEntryAssets.css)(() => stylesheetTag(clientEntryAssets.css)),
     ...ifElse(helmet)(() => helmet.style.toComponent(), []),
+    ifElse(process.env.BUILD_FLAG_IS_DEV === 'false')(() => inlineScript(webpackManifestScript)),
   ]);
```

Comment:
Hi, I believe this should be in the bodyElements array

Reply:
hey, could you give a reasoning for the same? i don't see much difference as long as the manifest is loaded before webpack entry files.

Reply:
Sorry, there is actually no difference, but I just thought to keep it consistent since other inlineScript is used inside the bodyElements.